interface/patient_file/pos_checkout_normal.php

```diff
@@ -211,44 +211,141 @@ function imdeleted() {
+            text-align: left;
+        }
+
+        table {
+            margin-top: 30px !important;
+            border-collapse: collapse;
+            width: 98%;
+        }
+
+        th,
+        td {
+            padding: 5px;
+        }
+
+        body > div:nth-child(2) > div:nth-child(3) > div > table > thead > tr{
+            background-color: var(--secondary);
+        }
+
+        body > div:nth-child(3) > div:nth-child(3) > div > table > thead{
+            background-color: var(--secondary);
+        }
+
+        body > div:nth-child(3) > div:nth-child(3) > div > table:nth-child(1) > tbody > tr:nth-child(3){
+            border: none!important;
+        }
+
+        .bg-blue{
+            background-color:var(--secondary);
+        }
+
+        .fac-name{
+            background-color:var(--secondary);
+            width: 99px;
+        }
+        .bg-color { background-color: var(--secondary); padding: 2px; font-weight: 600; -webkit-print-color-adjust: exact; }
+        
         </style>
         <title><?php echo xlt('Patient Checkout'); ?></title>
     </head>
     <body>
-        <div class="container mt-3">
-            <div class="row text-center">
-                <p class="font-weight-bold">
+    <div class="grid-container container mt-3" id="hideonprint, showonprint">
+            <div class="grid-child" style="padding-top:20px; font-weight:600;">
+                <p style="font-weight:600;">
+                    <bold class="bg-color"><?php echo text($frow['name']) ?></bold> <br /> <br />
                     <?php
-                    if ($GLOBALS['receipts_by_provider'] && !empty($providerrow)) {
-                        printProviderHeader($providerrow);
-                    } else {
-                        printFacilityHeader($frow);
-                    } ?>
-                    <?php
-                    echo xlt("Receipt Generated") . ":" . text(date(' F j, Y'));
-                    if ($invoice_refno) {
-                        echo " " . xlt("Invoice Number") . ": " . text($invoice_refno) . " " . xlt("Service Date")  . ": " . text($svcdate);
-                    }
+                        if ($GLOBALS['receipts_by_provider'] && !empty($providerrow)) {
+                            printProviderHeader($providerrow);
+                        } else {
+                            printFacilityHeader($frow);
+                        } ?>
+                        <?php
+                        echo xlt("Receipt Generated") . ":" . text(date(' F j, Y'));
+                        if ($invoice_refno) {
+                            echo " " . xlt("Invoice Number") . ": " . text($invoice_refno) . " " . xlt("Service Date")  . ": " . text($svcdate);
+                        }
                     ?>
                 </p>
             </div>
+            <div class="grid-child" style="padding-left:150px;">
+                <img src=<?php echo $GLOBALS['OE_SITE_WEBROOT'] . "/images/logo_1.png" ?> alt="" srcset="">
```

Comment:
suggest images are contained and responsive
class="img-fluid"
or 
max-width: ?; height: auto

also alt="" needs a value

Reply:
Ok. I'll make the changes

Reply:
hi @omegasystemsgroup, that commit, ad09d68d1b43d6075008bcc98da4d6596823125d, looks good in #5982 except for the styling, nice job. Can you bring that over here and then you can close that one?

Reply:
hello @stephenwaite, thanks.

But i am currently confused as to how to proceed with this. It's a bit new to me. So how do i bring the commit here?

Reply:
could just copy that file and then paste it into the file on this branch

Reply:
Alright. Done. I'll close the other one as suggested